### ISSUE TITLE
demangle C++ symbols/signatures

### ISSUE THREAD

**Issue by brendangregg** [iovisor/bpftrace#409]
bcc does this, it's really nice.

https://github.com/iovisor/bcc/commit/4ba52382f7ab0e42459ddbb09c79d25df03e4c2b
https://github.com/iovisor/bcc/commit/2f070516864928e3edd03425344685873944707d

bcc stackcount(8) printing a stack:

```
  Compile::Code_Gen()
  Compile::Compile(ciEnv*, C2Compiler*, ciMethod*, int, bool, bool, bool, DirectiveSet*)
  C2Compiler::compile_method(ciEnv*, ciMethod*, int, DirectiveSet*)
  CompileBroker::invoke_compiler_on_method(CompileTask*)
  CompileBroker::compiler_thread_loop()
  JavaThread::thread_main_inner()
  Thread::call_run()
  thread_native_entry(Thread*)
  start_thread
```

currently in bpftrace:

```
    _ZN7Compile8Code_GenEv+650
    _ZN7CompileC1EP5ciEnvP10C2CompilerP8ciMethodibbbP12DirectiveSet+3208
    _ZN10C2Compiler14compile_methodEP5ciEnvP8ciMethodiP12DirectiveSet+188
    _ZN13CompileBroker25invoke_compiler_on_methodEP11CompileTask+1016
    _ZN13CompileBroker20compiler_thread_loopEv+1352
    _ZN10JavaThread17thread_main_innerEv+446
    _ZN6Thread8call_runEv+376
    _ZL19thread_native_entryP6Thread+238
    start_thread+219
```

**Comment by caringi** [comment]
@brendangregg should the demangling be always activated or do you think it's better to be able to turn it off through a comand line option?

**Comment by brendangregg** [comment]
I think it should be enabled by default, but could be turned off. Maybe as an environment option (like https://github.com/iovisor/bpftrace/blob/master/docs/reference_guide.md#8-env-bpftrace_strlen ), eg, BPFTRACE_NO_CPP_DEMANGLE=1

**Comment by brendangregg** [comment]
fixed by #426
